src/components/containers/Popup.js

```diff
@@ -109,12 +110,12 @@ export class Popup extends Component {
     }
 
     render() {
-        const { className, name, padding, width = '80%', height = '80%', children } = this.props;
+        const { className, name, padding, width = '80%', height = '80%', children, onClose = this.hide } = this.props;
```

Comment:
I dont think this is the right way to modify it. onClose is a handler to the close event, but this makes it optional to close (which is questionable why you want this). So consider this instead

ie.

hide = () => {
  const { onClose } = this.props;
  popup.hide(this.props.name);
  if (onClose) onClose();
}